src/HotChocolate/Core/src/Types/Types/Descriptors/ObjectTypeDescriptor.cs

```diff
@@ -69,6 +69,21 @@ protected ObjectTypeDescriptor(IDescriptorContext context)
                 Definition.AttributesAreApplied = true;
             }
 
+            foreach (ObjectFieldDescriptor field in Fields)
+            {
+                if (!field.Definition.Ignore)
+                {
```

Comment:
I fixed the code style here, we always write the braces in C# code in order better see that

```csharp
if(foo is true)
    dosomething();
    dosomething();   
```

is really

```csharp
if(foo is true)
{
    dosomething();
}

dosomething();   
```

This issue quickly happens on refactorings and we decided to just enforce braces for if clauses.